servicecontrol/servicecontrol-instances/hardware.md

```diff
@@ -9,7 +9,8 @@ ServiceControl as an application can be used to process the entire message load
 ## General recommendations
 
 * Install ServiceControl on a dedicated server in production.
-* 6 GB of RAM minimum
+* Do not host ServiceControl and ServiceControl.Audit instance on the same server, host in isolation
+* 12 GB of dedicated RAM minimum per instance (16GB machine if only hosting ServiceControl).
```

Comment:
Not sure what this means, maybe break it down into two lines? e.g. when to use 12GB when to use 16GB?

Just a small improvement:

```suggestion
* A minimum of 12 GB of dedicated RAM per instance (16GB machine if only hosting ServiceControl).
```